### ISSUE TITLE
documentation website should use build from last tagged release, not current repo status

### ISSUE THREAD

**Issue by rlanzafame** [TeachBooks/TeachBooks#20]

**Comment by rlanzafame** [comment]
this will be automatic if we ensure that `main` is alwasy the most recent release and use `dev` branch for staging the next release. We should build the documentation site on the dev branch, and consider archiving major updates (once things become stable).

**Comment by BSchilperoort** [comment]
Currently the documentation is built locally and served with github pages.

Alternatively, you could use readthedocs for hosting the documentation. It's easily to set up and has git integration, but the biggest added value here is that they build docs per branch and per release. This makes it easy to view dev docs, stable docs, and docs of the older releases.
I can set this up if you'd like.

**Comment by rlanzafame** [comment]
yeah, that might be better...only downside that i can think of is the advertisements, which is fine. So if you want to do it, go for it!

I think @Tom-van-Woudenberg has already set up the readthedocs account with teachbooks (i am fuzzy on the details of how that works).

**Comment by BSchilperoort** [comment]
> only downside that i can think of is the advertisements, which is fine

I always forget about those because I use adblockers 😅 

We can disable paid ads though; https://docs.readthedocs.com/platform/stable/advertising/ethical-advertising.html#opting-out

**Comment by Tom-van-Woudenberg** [comment]
> I think [@Tom-van-Woudenberg](https://github.com/Tom-van-Woudenberg) has already set up the readthedocs account with teachbooks (i am fuzzy on the details of how that works).

I just did it for an example book, nothing more: https://app.readthedocs.org/projects/read-the-docs-example-book/

**Comment by BSchilperoort** [comment]
Docs are now on RTD including versions.

https://teachbooks.readthedocs.io/ points to "latest" which is the `develop` branch.

Any new release (i.e. tag) should also get a rendered docs version once they are generated. Older tags are missing the readthedocs yaml file so we can't build them on RTD.

Once the `stable` branch has the rtd yaml file, we can make that the default branch.

**Comment by rlanzafame** [comment]
i'll try to get PR #73 done tomorrow, or at least merge the part that is complete. Then i will update the readme and any other things to reflect the new setup!

**Comment by rlanzafame** [comment]
OK, this one is done since we are now using Read the Docs and I updated the README file with the link https://github.com/TeachBooks/TeachBooks/pull/73/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5
